client/src/pages/Home/index.js

```diff
@@ -1,5 +1,125 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
+import Axios from 'axios';
 
-const Home = () => <div>Hi from Home</div>;
+import Typography from '@material-ui/core/Typography';
+import { Button } from '@material-ui/core';
+import useMediaQuery from '@material-ui/core/useMediaQuery';
+import { useTheme } from '@material-ui/core/styles';
+
+import {
+  ButtonComponent,
+  ServiceCardForHomePage,
+  BlogsCard,
+  Footer,
+  Header,
+} from '../../component';
+
+import { BLOGS, BOOK } from '../../utils/router.constant';
+
+import useStyles from './style';
+
+const Home = () => {
+  const classes = useStyles();
+  const history = useHistory();
+  const theme = useTheme();
+
+  const isMobile = useMediaQuery(theme.breakpoints.down('xs'));
+
+  const [blogs, setBlogs] = useState([]);
+
+  const fetchData = async () => {
+    const { data } = await Axios.get('api/v1/blog');
+    setBlogs(data.data);
+  };
+  useEffect(() => {
+    fetchData();
+  }, []);
+
+  return (
+    <>
+      <Header />
+      <div className={classes.root}>
+        <div className={classes.firstSec}>
+          <div className={classes.intro}>
+            <Typography color="primary" variant="h4" className={classes.text}>
+              Endless looks. We are committed to create the perfect style for
+              our clients.
+            </Typography>
+            <Typography variant="h6" className={classes.text2}>
+              Best brands of products. we will deliver products to your house,
+              Shopping is much easier now!!
+            </Typography>
+            <ButtonComponent
+              variant="outlined"
+              color="primary"
+              className={classes.button}
+              onClick={() => history.push(BOOK)}
+            >
+              BookNow
+            </ButtonComponent>
+            <Button
+              variant="outlined"
+              color="contained"
```

Comment:
button in material UI don't have property color
please remove this line because it shows an error in the console.
![Uploading image.png…]()